qrenderdoc/3rdparty/catch/official/catch.hpp

```diff
@@ -20,6 +20,7 @@
   actual site failure. So the debugger stops on the CHECK() or REQUIRE()
   call instead of inside AssertionHandler::complete()
 * https://github.com/baldurk/renderdoc/commit/4232736fc21fc6a13a4de6997a5ae106598b225f
+* Add FreeBSD support with correct debugger handling
```

Comment:
I'd rather you didn't make any modifications to upstream third party files. Instead you should disable the unit tests completely on your platform so this file isn't included - I believe they're already disabled on Android so you can follow the same pattern.

Reply:
About disabling unit tests: If add new platform to globalconfig.h, code needs more `|| ENABLED(RDOC_BSD)` in many places so I simply add `|| defined(__FreeBSD__)`